providers/implementations/ciphers/ciphercommon.c

```diff
@@ -150,6 +150,7 @@ static int cipher_generic_init_internal(PROV_CIPHER_CTX *ctx,
                                         int enc)
 {
     ctx->enc = enc ? 1 : 0;
+    ctx->num = 0;
```

Comment:
Could you please swap lines 152 and 153, to make the code change consistent with the other two?